### ISSUE TITLE
placesCenter@scollins: Memory leaks and Cinnamon crashes

### ISSUE THREAD

**Issue by BeatLink** [linuxmint/cinnamon-spices-applets#6675]
### Applet version/Build date

2.4

### Cinnamon version

6.2.9

### Distribution

Mint 22

### Graphics hardware *and* driver used

NVIDIA Corporation GA107M [GeForce RTX 3050 Ti Mobile] and nvidia-driver-550

### Applet name and maintainer

placesCenter@scollins @collinss

### What happened?

When placesCenter is enabled and files are deleted, Cinnamon's memory usage rises rapidly. If enough files are deleted, cinnamon inevitably crashes

**Comment by claudiux** [comment]
Solved with `Direct@claudiux`: https://cinnamon-spices.linuxmint.com/applets/view/422

**Comment by BeatLink** [comment]
Very nice. Any plans to merge with or replace upstream?

**Comment by claudiux** [comment]
No, I haven't received any response from the maintainer of the upstream applet.

So I created this applet, and I am free to manage its code.
